pubspec.lock

```diff
@@ -261,6 +266,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "4.0.0"
+  intl:
+    dependency: "direct main"
+    description:
+      name: intl
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "0.17.0"
```

Comment:
@luongvo Same above, do we only need to configure as : `intl: ^0.17.0` ?

Reply:
pub.lock is auto generated with the updated dep from pubspec.yaml @doannimble 😂

Reply:
@sleepylee ahhh, then it should be okay 😆